03app_edhoc/03app_edhoc.c

```diff
@@ -113,7 +113,7 @@ int main(void) {
     }
 
     if(!res) {
-      res = initiator_prepare_message_3(&initiator_processed_m2, ByReference, NULL, &initiator_done, &message_3, prk_out);
+      res = initiator_prepare_message_3(&initiator_processed_m2, ByReference, NULL, &initiator_done, &message_3, &prk_out);
```

Comment:
same here, where does this change come front, it has nothing to do with your issue